### ISSUE TITLE
Missing import in mast/services.py in astroquery 0.4.1

### ISSUE THREAD

**Issue by odysseus9672** [astropy/astroquery#1764]
Line 207 in the file `mast/services.py` is
```
            warnings.warn("Query returned no results.", NoResultsWarning)
```
when the code reaches this line it leads to an error:
```
NameError: name 'warnings' is not defined
```

`services.py` is missing the `import warnings` line that was in `mast/core.py` for `astroquery` 0.4.

**Comment by DoctorCronut** [comment]
Hi @odysseus9672, I'm a university student new open source contributions. Can I pick this one up?

**Comment by odysseus9672** [comment]
@DoctorCronut Sure. I don't care, as long as it gets fixed.

**Comment by bsipocz** [comment]
@DoctorCronut - 👋 thanks for your interest. #1768 already fixes this issue, so I would suggest to browse the issues for something else to fix. We have a long backlog, so even old issues are still relevant. If you have any question about contributing feel free to ask, or consult the docs we have on contributing (the docs certainly is not complete, so indeed feel free to ask): https://docs.astropy.org/en/latest/development/workflow/development_workflow.html

**Comment by keflavich** [comment]
Closed by #1768
